src/components/MDXComponents.tsx

```diff
@@ -52,65 +31,57 @@ const MDXComponents = {
   // headings
   h1: ({ children }: any) => {
     return (
-      <Heading as='h1' textAlign='start' fontSize='4xl' mb={5}>
+      <Heading as='h1' textAlign='start' mb={5} {...header1}>
         {children}
       </Heading>
     );
   },
   h2: ({ children }: any) => {
     return (
-      <Heading as='h2' textAlign='start' fontSize='3xl' mb={4}>
+      <Heading as='h2' textAlign='start' mt='16 !important' mb={4} {...header2}>
         {children}
       </Heading>
     );
   },
   h3: ({ children }: any) => {
     return (
-      <Heading as='h3' fontSize='2xl' mt={5} mb={2.5}>
+      <Heading as='h3' mt={5} mb={2.5} {...header3}>
         {children}
       </Heading>
     );
   },
   h4: ({ children }: any) => {
     return (
-      <Heading as='h4' fontSize='lg' mb={2.5}>
+      <Heading as='h4' mb={2.5} {...header4}>
         {children}
       </Heading>
     );
   },
+  // tables
+  table: ({ children }: any) => (
+    <Flex maxW='min(100%, 100vw)' overflowX='scroll'>
+      <Table
```

Comment:
table needs a bigger margin-bottom <img width="522" alt="Screen Shot 2022-11-28 at 13 36 56" src="https://user-images.githubusercontent.com/948922/204347637-ec179a00-a6e5-43f4-85bb-c3d3ca2dc73f.png">

Reply:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/54227730/204352935-9d0dba87-0af6-47c1-808e-d336ca1cd5ef.png">
Not sure how much more we want... this looks pretty close to the Figma. @nloureiro Maybe another 2px?

Reply:
Are you referencing to the margin-bottom of the table?

Reply:
@nloureiro Yep. Nico mentioned it looks like it needs a bit more margin on the bottom, but the Figma looks about the same. Did you want to have more space underneath markdown tables?

Reply:
if yes we now have this:
![Screen Shot 2022-11-28 06 34 26 PM](https://user-images.githubusercontent.com/1120748/204354188-5e6e3b00-3fbd-440b-8000-4a2d54294c05.png)

might change to this:
![Screen Shot 2022-11-28 06 35 05 PM](https://user-images.githubusercontent.com/1120748/204354297-d5a6cf36-ee81-42e3-982e-043d7dc64361.png)


what do you think? small detail....

Reply:
Looks good, just pushed a commit to switch to use `6` as the bottom margin instead of `4`